srcs/execution/builtins/echo.c

```diff
@@ -12,25 +12,29 @@
 int		builtin_echo(int argc, char **argv)
 {
 	int		i;
-	t_opt	*o;
+	char		*opt;
 
-	o = (t_opt*)memalloc_or_die(sizeof(t_opt));
-	i = retrieve_options(argc, argv, o, "sne");
-	while (++i < argc)
+	if ((opt = get_options_core(argc, argv)) == (char *)-1)
+		return (STATUS_FAILURE);
+	i = 1;
+	while (i < argc && argv[i][0] == '-' && argv[i][1] != '-')
```

Comment:
I am looking at export builtin and I couldn't understand what I am missing.
What should I do?
Or do it.